widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -173,6 +176,9 @@ public void error(GliaException ex) {
     private final RemoveFileAttachmentUseCase removeFileAttachmentUseCase;
     private final IsShowSendButtonUseCase isShowSendButtonUseCase;
     private final IsShowOverlayPermissionRequestDialogUseCase isShowOverlayPermissionRequestDialogUseCase;
+    private final DownloadFileUseCase downloadFileUseCase;
+
+    private Disposable disposable = null;
```

Comment:
Where do we dispose it?

Reply:
onDestroy - forgot to add :D now its here